deploy/kubernetes/base/node.yaml

```diff
@@ -15,21 +15,14 @@ spec:
       # Host network must be used for interaction with Workload Identity in GKE
       # since it replaces GCE Metadata Server with GKE Metadata Server. Remove
       # this requirement when issue is resolved and before any exposure of
-      # metrics ports.
-      hostNetwork: true
+      # metrics ports. But hostNetwork is not working for Windodws, might be an issue
```

Comment:
typo: windows

Do we support workload identity on Windows? If so, then not being on host network may be problematic.

Reply:
fixed. we might use a different way to workaround it. Currently we don't need to set it for windows.